forecasting/src/autogluon/forecasting/trainer/auto_trainer.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from typing import Dict, Union, Optional, Any
+
+from ..models.presets import get_preset_models
+from .abstract_trainer import AbstractForecastingTrainer, TimeSeriesDataFrame
+
+logger = logging.getLogger(__name__)
+
+
+class AutoForecastingTrainer(AbstractForecastingTrainer):
+    def construct_model_templates(self, hyperparameters, **kwargs):
+        path = kwargs.pop("path", self.path)
+        eval_metric = kwargs.pop("eval_metric", self.eval_metric)
+        hyperparameter_tune = kwargs.get("hyperparameter_tune", False)
+        return get_preset_models(
+            path=path,
+            eval_metric=eval_metric,
+            prediction_length=self.prediction_length,
+            freq=self.freq,
+            hyperparameters=hyperparameters,
+            hyperparameter_tune=hyperparameter_tune,
+            quantiles=self.quantiles,
+            use_feat_static_cat=self.use_feat_static_cat,
+            use_feat_static_real=self.use_feat_static_real,
+            cardinality=self.cardinality,
+        )
+
+    # todo: implement cross-validation / holdout strategy
+    # todo: including CVSplitter logic
+    def fit(
+        self,
+        train_data: TimeSeriesDataFrame,
```

Comment:
Will we have a convenient function in this class to convert TimeSeriesDaaFrame to ListDataset?